### PR TITLE
Change to zos ftp for zowe cli and update readme

### PR DESCRIPTION
**Pull request by std4lqi**
1) Changed the profile setting secureFtp to true by default.
2) Changed plugin name from "zos-ftp" to "zos-ftp-for-zowe-cli".
3) Commit README.md with the sign-off, which is updated in https://github.com/std4lqi/zowe-cli-ftp-plugin/blob/update-readme/README.md.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@zowe/zos-ftp",
-  "version": "0.1.0",
+  "name": "@brightside/zos-ftp-for-zowe-cli",
```
Comment (zFernand0) [inline package.json]: I believe we can keep the name with an `@zowe/` scope since this is a contributed plugin. : )

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "1.0.0",
```
Comment (zFernand0) [inline package.json]: Before going to a 1.0.0 I would like to run the CLI commands by some UX people (unless you already did this) : )

File: package.json (hunk 1 above)
Comment (std4lqi) [reply package.json]: I changed it to `@zowe/zos-ftp-for-zowe-cli` now. :-)

File: package.json (hunk 2 above)
Comment (std4lqi) [reply package.json]: I change it back to `0.1.0`.

**Pull request merged**